src/main.cpp

```diff
@@ -247,6 +260,27 @@ int init_options(int argc, char**argv, struct options& options)
       usage();
       return -1;
 
+    case MEMCACHED_WRITE_FORMAT:
+      if (strcmp(optarg, "binary") == 0)
+      {
+        LOG_INFO("Memcached write format set to 'binary'");
+        options.memcached_write_format = MemcachedWriteFormat::BINARY;
+      }
+      else if (strcmp(optarg, "json") == 0)
+      {
+        LOG_INFO("Memcached write format set to 'json'");
+        options.memcached_write_format = MemcachedWriteFormat::JSON;
+      }
+      else
+      {
+        LOG_WARNING("Invalid value for memcached-write-format, using '%s'."
+                    "Got '%s', valid vales are 'json' and 'binary'",
+                    ((options.memcached_write_format == MemcachedWriteFormat::JSON) ?
```

Comment:
I'm not sure I understand. I am printing two things: the setting we are using, and the option that was received. Can you elaborate?